src/sbm.jl

```diff
@@ -1049,3 +1052,35 @@ function update_after_subsurfaceflow(sbm::SBM, zi, exfiltsatwater)
         sbm.zi[i] = zi[i]
     end
 end
+
+function update_total_water_storage(sbm::SBM, rnet, rrouting, lrouting)
+
+    # Set the total storage to zero
+    fill!(sbm.total_storage, 0)
+
+    # Burn the river routing values
+    sbm.total_storage[rnet] = (
+        rrouting.h_av .* sbm.riverfrac[rnet] * 0.001 # convert to mm
```

Comment:
Just realized that the use of `riverfrac` (as we discussed) is not always correct, as a river area can be larger than the grid cell area. I think it is better to first compute the river volume (based on `h_av`) and then divide by river area.

Reply:
Good point, adjusted that.